requirements.txt

```diff
@@ -20,5 +20,5 @@ requests==2.32.3
 rsa==4.7.2  # resolve ambiguity with build-time dependency
```

Comment:
Did you try updating this? What are the version constraints?

Reply:
Didn't 😅, https://github.com/DataBiosphere/azul/issues/6774.